bin/run_fs_recon.py

```diff
@@ -122,10 +121,11 @@ def _run(subjects_dir, subject, raw_dir, force, mp, volume):
     run_subprocess(['mri_concat', '--rms', '--i', mri,
                     '--o', op.join(subjects_dir, subject, 'mri/orig/001.mgz')],
                    env=this_env)
-    run_subprocess(['recon-all', '-openmp', mp, '-subject', subject, '-all'], env=this_env)
+    run_subprocess(['recon-all', '-openmp', '%.0f' % mp, '-subject', subject, '-all'], env=this_env)
```

Comment:
Thanks @Eric89GXL I actually tried several variants to get this to work, and this was the one that `run_subprocess` appeared to like. Will keep it in mind and let you know if I encounter `TypeErrors` again.